libs/response/src/lib/mocks/sport-event-tv-search.mock.ts

```diff
@@ -30,6 +30,7 @@ export const sportEventTvSearchMock: TvSearchResult = {
         sport: 'Basketball',
         category: 'USA',
         tournament: 'NBA',
+        league: 'NBA G-League',
```

Comment:
G-League is a separate league. For NBA expected values are: league='NBA', tournament='NBA, Regular' or tournament='NBA Preseason' or tournament='All Star Game'.